kafka_consumer/tests/conftest.py

```diff
@@ -106,33 +145,40 @@ def kafka_instance():
 def create_topics():
     client = _create_admin_client()
 
+    if set(TOPICS).issubset(set(client.list_topics(timeout=1).topics.keys())):
+        return True
+
     for topic in TOPICS:
-        client.create_topics([NewTopic(topic, 2, 1)], operation_timeout=1, request_timeout=1)
+        client.create_topics([NewTopic(topic, 2, 1)])
+        time.sleep(1)
+
+    return set(TOPICS).issubset(set(client.list_topics(timeout=1).topics.keys()))
+
 
-    return set(client.list_topics(timeout=1).topics.keys()) == set(TOPICS)
+def wait_for_cp_kafka_topics():
```

Comment:
Would this also be useful for non-kerberos environments? Or are these topics only found in kerberos?